model/src/main/kotlin/utils/Extensions.kt

```diff
@@ -48,7 +48,7 @@ fun Package.toClearlyDefinedTypeAndProvider(): Pair<ComponentType, Provider>? =
         "Crate" -> ComponentType.CRATE to Provider.CRATES_IO
         "DotNet", "NuGet" -> ComponentType.NUGET to Provider.NUGET
         "Gem" -> ComponentType.GEM to Provider.RUBYGEMS
-        "GoDep", "GoMod" -> ComponentType.GO to Provider.GOLANG
+        "Go" -> ComponentType.GO to Provider.GOLANG
```

Comment:
I believe e30fab04d353d7fd8163aded60c8d1dad00dbdbc is not the commit you intended to reference, but another one which is IIRC very close to it timely.

Reply:
Thanks, fixed by referencing the right commits (it's actually two).